vnext/Microsoft.ReactNative.Cxx.UnitTests/ReactContextTest.cpp

```diff
@@ -20,18 +20,30 @@ struct ReactContextStub : implements<ReactContextStub, IReactContext> {
   }
 
   void CallJSFunction(
-      hstring const & /*moduleName*/,
-      hstring const & /*functionName*/,
-      JSValueArgWriter const & /*paramsArgWriter*/) noexcept {
-    VerifyElseCrashSz(false, "Not implemented");
+      hstring const &moduleName,
+      hstring const &functionName,
+      JSValueArgWriter const &paramsArgWriter) noexcept {
+    Module = moduleName;
+    Method = functionName;
+    auto writer = MakeJSValueTreeWriter();
+    paramsArgWriter(writer);
+    Args = TakeJSValue(writer);
   }
 
   void EmitJSEvent(
-      hstring const & /*eventEmitterName*/,
-      hstring const & /*eventName*/,
-      JSValueArgWriter const & /*paramsArgWriter*/) noexcept {
-    VerifyElseCrashSz(false, "Not implemented");
+      hstring const &eventEmitterName,
+      hstring const &eventName,
+      JSValueArgWriter const &paramsArgWriter) noexcept {
+    Module = eventEmitterName;
+    Method = eventName;
+    auto writer = MakeJSValueTreeWriter();
+    paramsArgWriter(writer);
+    Args = TakeJSValue(writer);
   }
+
+  std::wstring Module;
```

Comment:
consider naming: 
- we've been using m_camelCase
- include Name in the name so e.g. m_moduleName

Reply:
We usually use the m_camelCase for private fields. In this case for simplicity I use public fields and it is the reason for the Pascal case. https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions